src/components/Sidebar.tsx

```diff
@@ -36,9 +36,19 @@ const LogoContainer = styled.div`
   position: relative;
 `;
 
-const LogoImage = styled.img`
+const LogoBackground = styled.div`
   height: 50px;
   width: 50px;
+  background-image: url(${BackgroundTexture});
+  position: absolute;
+  left: 0;
+  z-index: 1;
+  transform: translateX(-1%);
+`;
+
+const LogoImage = styled.img`
+  height: 50px;
+  width: 54px;
```

Comment:
might be easier to just target the img in the LogoBackground div rather than creating a new styled component for it.